apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/tasks/OperationTasksBuilder.java

```diff
@@ -71,6 +73,26 @@ private List<OperationTask<?>> prepareDataLayoutOperationTaskList(JobConf.JobTyp
         selectedStrategyIndices.stream()
             .map(tableDataLayoutMetadataList::get)
             .collect(Collectors.toList());
+    double totalComputeCost =
+        selectedTableDataLayoutMetadataList.stream()
+            .map(m -> m.getDataLayoutStrategy().getCost())
+            .reduce(0.0, Double::sum);
+    double totalReducedFileCount =
+        selectedTableDataLayoutMetadataList.stream()
+            .map(m -> m.getDataLayoutStrategy().getGain())
+            .reduce(0.0, Double::sum);
+    log.info(
+        "Total estimated compute cost: {}, total estimated reduced file count: {}",
+        totalComputeCost,
+        totalReducedFileCount);
+    meter
+        .counterBuilder("data_layout_optimization_compute_cost")
```

Comment:
```suggestion
        .counterBuilder("data_layout_optimization_estimated_compute_cost")
```

Reply:
I don't think we have metric for compute cost, YARN/Spark team might have some metrics for this, need to explore. The DC app reports these metrics https://github.com/linkedin/openhouse/blob/32964d11ab46ffd5b5d41b2efa71a1a060641fc5/apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/DataCompactionSparkApp.java#L61, specifically rewritten file count and added file count, the diff is actual reduced file count.

Reply:
perfect.